Draft abstract to be submitted (#2)
11michalis11/Talks: THIS Event 2020/Abstract.tex

\documentclass{article}

\title{A game theoretic model of the behavioural gaming that takes place at the EMS - ED interface}
\author{Michalis Panayides}

\begin{document}

\maketitle
\begin{abstract}

>     Mathematical modelling can be utilised to model a vast range of scenarios applied to areas like healthcare, supermarkets and many more. Although, such methods are really powerful, they sometimes lack the possible insight that one may get by considering more qualitative approaches. The \textit{mother} of such approaches is considered by many to be ethnography. 

Reviewer: ```suggestion
    Mathematical modelling can be utilised to model a vast range of scenarios applied to areas like healthcare, supermarkets and more. 
    Although, such methods are powerful, they sometimes lack the possible insight that one may get by considering more qualitative approaches. The \textit{precursor} of such approaches is considered by many to be ethnography. 
```

Line break your tex files (I have a strict 80 character line break setup on my editor for all files). This ensures that you diffs are more meaningful.

>     Mathematical modelling can be utilised to model a vast range of scenarios applied to areas like healthcare, supermarkets and many more. Although, such methods are really powerful, they sometimes lack the possible insight that one may get by considering more qualitative approaches. The \textit{mother} of such approaches is considered by many to be ethnography. 

Reviewer 2: Operational Research (OR) techniques provide a toolkit of mathematical modelling approaches that are routinely used for problem solving in many sectors, increasingly so in healthcare.

>     Ethnographic studies have been around since the \(19^{th}\) century and their main ideology is that a researcher that aims to study a specific social setting should acquire a deeper understanding of a culture's norms and values. The main technique that accompanies ethnography is participant observation, where the researcher participates in the social setting they wish to study and also records any observations made on it. Two additional concepts of ethnography are the ideas of emic models and etic models. An etic model is built based solely on the knowledge of the modeller and how they perceive the model. On the contrary an emic model takes into consideration the perspective of the individuals that belong in the social setting of interest. Ethnography suggests that an emic model should always be the default type of model used when studying any kind of social setting.

Reviewer: ```suggestion
    Ethnographic studies have been around since the \(19^{th}\) century and their guiding methodology is that a researcher that aims to study a specific social setting should acquire a deeper understanding of the cultural norms and values. The main technique that accompanies ethnography is participant observation, where the researcher participates in the social setting they wish to study recording their observations.
```

>     A great application of these ideas and principles is the emergent behaviour that takes place on the interface between Emergency Medical Services (EMS) and the Emergency Department (ED). Numerous decisions are taken by both patients and staff alike that determine the level of workflow and the patient pathway daily. There is empirical evidence to suggest that imposing targets in the ED results in gaming at the interface of care between the EMS and ED. Multiple scenarios are examined where an ambulance service needs to distribute patients between neighbouring hospitals. The interaction between the hospitals and the ambulance service is modelled in a game theoretic framework where the ambulance service has to decide how many patients to distribute to each hospital.

Reviewer: ```suggestion
    The application of these ideas and principles considered here is the emergent behaviour that takes place at the interface between Emergency Medical Services (EMS) and the Emergency Department (ED). Numerous decisions are taken by both patients and staff alike that determine the level of workflow and the patient pathway. There is empirical evidence to suggest that imposing targets in the ED results in gaming at the interface of care between the EMS and ED. Multiple scenarios are examined where an ambulance service needs to distribute patients between neighbouring hospitals. The interaction between the hospitals and the ambulance service is modelled in a game theoretic framework, supported by a novel Markov model, where the ambulance service has to decide how many patients to distribute to each hospital.
    
This work brings together ethnography and game theory to understand the emergent behaviour in healthcare settings.
```

\documentclass{article}

\title{A game theoretic model of the behavioural gaming that takes place at the EMS
 - ED interface}
\author{Michalis Panayides}

\begin{document}

\maketitle
\begin{abstract}
    Operational Research (OR) techniques provide a toolkit of mathematical 
    modelling approaches that are routinely used for problem solving in many
    sectors. Recent research shows that behavioural studies may accompany such
    techniques and offer some additional insight to the problem studied and 
    thus enrich some of the already existing traditional methods. Ethnography is 

>     considered by many to be one of the main such approaches.

Reviewer: ```suggestion
    considered to be one of the main such approaches.
```

    Ethnographic studies have been around since the \(19^{th}\) century and
    their main guiding methodology is that a researcher that aims to study 
    a specific social setting should acquire a deeper understanding of the 
    cultural norms and values. The main technique that accompanies ethnography 
    is participant observation, where the researcher participates in the social 
    setting they wish to study recording their observations.

    The application of these ideas and principles considered here is the emergent 
    behaviour that takes place at the interface between Emergency Medical Services 
    (EMS) and the Emergency Department (ED). Numerous decisions are taken by both 
    patients and staff alike that determine the level of workflow and the patient 
    pathway. There is empirical evidence to suggest that imposing targets in the ED 

>     results in gaming at the interface of care between the EMS and ED. Multiple 
>     scenarios are examined where an ambulance service needs to distribute patients 

Reviewer: ```suggestion
    results in gaming at the interface of care between the EMS and ED. 
    
    In this work, multiple 
    scenarios are examined where an ambulance service needs to distribute patients 
```
